samza-api/src/main/java/org/apache/samza/system/descriptors/GenericInputDescriptor.java

```diff
@@ -40,6 +40,10 @@
     super(streamId, serde, systemDescriptor, null);
   }
 
+  GenericInputDescriptor(String streamId, SystemDescriptor systemDescriptor, Serde serde, InputTransformer transformer) {
```

Comment:
This shouldn't be here. There was a conscious decision to not support stream level transformers. If this being used would like to understand the use case.

Reply:
rolled back